docs/60_Usage_Best_Practices.md

```diff
@@ -0,0 +1,300 @@
+|---------------------------|--------------------------------|------------------------------------------------|----------------------------------------------------------------------------------|------------------------|
+| PREVENTS DATA DUPLICATION |             &Cross;            |                     &Cross;                    |                                      &check;                                     |         &check;        |
+| PREVENTS DATA LOSS        |             &Cross;            |                     &Cross;                    |                                      &Cross;                                     |         &check;        |
+
+Now let's examine in detail the benefits and drawbacks of the different procedures listed.
+
+### Pagination - unordered:
+
+#### Benefits
+
+- It allows retrieving data in smaller chunks, which can be useful for displaying data in a paginated manner.
+- It provides control over the number of records fetched per page, optimizing resource utilization.
+- It can handle large result sets efficiently by retrieving only a subset of data at a time.
+
+#### Drawbacks
+
+- The order of records may not be guaranteed unless additional sorting criteria are applied.
+- If the data is frequently updated or deleted, the pagination result may become inconsistent.
+
+### Pagination - ordered:
+
+#### Benefits
+
+- It retrieves data in a specific order, providing consistency in the result set.
+- It is useful when maintaining a consistent view of data is crucial, such as in chronological or alphabetical sorting.
+
+#### Drawbacks
+
+- It may introduce performance overhead when retrieving large result sets, especially if sorting is complex.
+- If the data changes during pagination, it may lead to missing or duplicate records on subsequent pages.
+
+### Stream
+
+#### Benefits
+
+- It provides real-time data streaming, allowing applications to consume data as it becomes available.
+- It enables efficient processing and analysis of continuous data streams.
+- It can be useful in scenarios where immediate data updates are critical, such as real-time analytics or monitoring.
+
+#### Drawbacks
+
+- It requires continuous processing and handling of data streams, which may increase resource utilization.
+- Data ordering may not be guaranteed, depending on the stream source and processing pipeline.
+- In the event of disruptions or failures in stream processing, data consistency can be compromised.
+
+:::tip
+It's important to choose the appropriate procedure based on the specific requirements and trade-offs of your application.
+:::
+
+## Scenarios
+
+Let's consider the case of a single view called `my_single_view`, which corresponds to a MongoDB collection associated with the following JSON definition by the CRUD Service:
+
+```json
+{
```

Comment:
Should we also add the other properties of collection definition, so that one can just copy this and try it out locally?

Reply:
Hi @danibix95, I've updated the collection definition to match the whole schema accepted by the crud-service

Reply:
Hi, I think it is still missing the `endpointBasePath` property 🤔